src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/services/GoalService.kt

```diff
@@ -143,8 +140,8 @@ class GoalService(
   }
 
   @Transactional
-  fun deleteGoal(goalUuid: UUID): Unit? {
-    val goalEntity: GoalEntity? = goalRepository.findByUuid(goalUuid)
-    return goalEntity?.let { goalRepository.delete(it) }
+  fun deleteGoalByUuid(goalUuid: UUID): Int {
+    val goalsDeletedCount = goalRepository.deleteByUuid(goalUuid)
```

Comment:
just `return goalRepository.deleteByUuid(goalUuid)`

Reply:
ah yes, this is left in from when I was doing some debugging.

Reply:
https://github.com/ministryofjustice/hmpps-sentence-plan/pull/104/commits/35c138c3305c49a6e5495fbdba9018d327b2cba8